lua/dashboard/init.lua

```diff
@@ -182,13 +182,15 @@ function db:load_theme(opts)
     config = vim.tbl_extend('force', config, self.opts.preview)
   end
 
-  require('dashboard.theme.' .. opts.theme)(config)
+  local theme = require('dashboard.theme.' .. opts.theme)
+  theme.init(config)
```

Comment:
does these will increase  startup time ?. header  use auto pad is enough like what i do. it add pad in callback after all content insert to buffer. then insert some `''` to the buffer head.

Reply:
Padding now only gets added once, whereas with the callbacks the header has to be regenerated with each callback.

As for the startup time, there is no difference (I think) for each method.  Both has to generate the header with padding at startup.

Reply:
I still only insert `''` as you requested, I am just making sure it only happens once.

Reply:
okay . I will test it later.

Reply:
okay . I will test it later.